build.gradle

```diff
@@ -24,6 +25,28 @@ dependencies {
 	testCompile 'junit:junit:4.12'
 }
 
+license {
+	include "**/*.java"
+    header rootProject.file('utils/licenseHeader')
+    headerDefinitions {
+        custom_definition {
+          firstLine = "/*******************************************************************************"
+          endLine   = " *******************************************************************************/"
+          beforeEachLine = " * "
+          firstLineDetectionPattern = "/*"
+          lastLineDetectionPattern  = " *"
+          allowBlankLines = false
+          isMultiline = true
+        }
+    }
+    mapping {
+    	java='custom_definition'
+    }
+    strictCheck true
+}
+
+build.dependsOn licenseTest
```

Comment:
Does also check.dependsOn(...) work? Ideally, you should be able to successfully build the project but check (which also includes unit and integration tests) fails in case of missing headers.

Reply:
Thanks for the hint. Of course, it's part of check now.